code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -68,6 +68,7 @@
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR|HIDESNOUT
 	flags_cover = HEADCOVERSMOUTH|HEADCOVERSEYES|PEPPERPROOF
 	visor_flags_inv = HIDEEYES|HIDEFACE
+	slowdown = 0
```

Comment:
I think what you could do to offset this change is to make it so that plasmamen helmets don't provide insulation.

So that if they want full space immunity, they should wear something like an actual space helmet or something that provides that level of protection.

Take a look at the `min_cold_protection_temperature` and `max_heat_protection_temperature` variables to achieve this

Reply:
```suggestion
	slowdown = 0
	min_cold_protection_temperature = HELMET_MIN_TEMP_PROTECT
	max_heat_protection_temperature = HELMET_MAX_TEMP_PROTECT
```

Reply:
> So that if they want full space immunity, they should wear something like an actual space helmet...

This is a bad idea IMO.

The plasmamen helmet has always been their safety net against being fully stripped and burning to death/husk.
They are the species with the most recognizable equipment for a good reason, they need it and we dodge a lot of bad interactions between players because we have kept it consistent over the years.

Reply:
do normal space helmets/suits prevent plasmaman ignition?

Reply:
> do normal space helmets/suits prevent plasmaman ignition?

No